flycheck-pos-tip.el

```diff
@@ -65,8 +66,31 @@ The function should be a no-op in this case."
   :type 'function
   :package-version '(flycheck-pos-tip . "0.2"))
 
+(defcustom flycheck-pos-tip-tty-show-function
+  #'flycheck-pos-tip-tty-show
+  "A function to show messages on TTY frames.
+
+This function is called instead of
+`flycheck-pos-tip-show-function' on TTY frames."
+  :group 'flycheck-pos-tip
+  :type 'function
+  :package-version '(flycheck-pos-tip . "0.3"))
+
+(defcustom flycheck-pos-tip-tty-hide-function
+  #'identity
```

Comment:
`ignore` is the nicer default here, I think…  it doesn't make any difference in behaviour, but it just reads nicer in my opinion.